tests/integration_tests_control_files/advection_explicit_periodic/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
-configure_file(2D_advection_explicit_periodic.prm 2D_advection_explicit_periodic.prm COPYONLY)
+configure_file(2D_advection_explicit_periodic_energy.prm 2D_advection_explicit_periodic_energy.prm COPYONLY)
 add_test(
- NAME MPI_2D_ADVECTION_EXPLICIT_PERIODIC_LONG
-COMMAND mpirun -n ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_2D -i ${CMAKE_CURRENT_BINARY_DIR}/2D_advection_explicit_periodic.prm
+ NAME MPI_2D_ADVECTION_EXPLICIT_PERIODIC_ENERGY_LONG
+COMMAND mpirun -n ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_2D -i ${CMAKE_CURRENT_BINARY_DIR}/2D_advection_explicit_periodic_energy.prm
   WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 )
+set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
+configure_file(2D_advection_explicit_periodic_OOA.prm 2D_advection_explicit_periodic_OOA.prm COPYONLY)
+add_test(
+ NAME MPI_2D_ADVECTION_EXPLICIT_PERIODIC_OOA_LONG
```

Comment:
use full name instead of "OOA"

Reply:
I left OOA since the header and cpp files explained OOA as orders of accuracy